ocs_ci/deployment/acm.py

```
@@ -188,8 +191,81 @@ def deploy_downstream(self):
+
+            # 2. Switch to the managed cluster and wait for the routeagent DaemonSet rollout
+            config.switch_ctx(cluster_index)
+            kubeconfig = cluster.RUN["kubeconfig"]
+            rollout_cmd = (
+                f"oc --kubeconfig {kubeconfig} rollout status "
+                f"daemonset/submariner-routeagent "
+                f"-n {constants.SUBMARINER_OPERATOR_NAMESPACE} --timeout=120s"
+            )
+            try:
+                exec_cmd(rollout_cmd)
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)